src/frontend/src/pages/HomePage/WorkPackagesByTimelineStatus.tsx

```diff
@@ -16,8 +16,41 @@ import { routes } from '../../utils/routes';
 import LoadingIndicator from '../../components/LoadingIndicator';
 import PageBlock from '../../layouts/PageBlock';
 import ErrorPage from '../ErrorPage';
-import { FormControl, InputLabel, MenuItem, Select, Typography, useTheme } from '@mui/material';
-import DetailDisplay from '../../components/DetailDisplay';
+import {
+  Chip,
+  Grid,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  Typography,
+  useTheme,
+  CircularProgress
+} from '@mui/material';
+import { Construction, Work } from '@mui/icons-material';
+import { CircularProgressProps } from '@mui/material/CircularProgress';
+
+function CircularProgressWithLabel(props: CircularProgressProps & { value: number }) {
```

Comment:
probably can make this into a component. const Circular... = (props ...) => { ... }